securedrop/models.py

```diff
@@ -118,13 +118,18 @@ def public_key(self):
     def to_json(self):
         docs_msg_count = self.documents_messages_count()
 
+        if(self.last_updated):
```

Comment:
I think the linter is going to to yell at this for not being `if self.last_updated:` (no parenthesis).